webapps-backup/src/main/java/io/camunda/webapps/backup/repository/elasticsearch/ElasticsearchBackupRepository.java

```diff
@@ -237,30 +249,43 @@ public void executeSnapshotting(
       final Runnable onSuccess,
       final Runnable onFailure) {
     final var request =
-        new CreateSnapshotRequest()
-            .repository(snapshotRequest.repositoryName())
-            .snapshot(snapshotRequest.snapshotName())
-            .indices(snapshotRequest.indices())
-            // ignoreUnavailable = false - indices defined by their exact name MUST be present
-            // allowNoIndices = true - indices defined by wildcards, e.g. archived, MIGHT BE absent
-            .indicesOptions(IndicesOptions.fromOptions(false, true, true, true))
-            .includeGlobalState(backupProps.includeGlobalState())
-            .userMetadata(
-                objectMapper.convertValue(snapshotRequest.metadata(), new TypeReference<>() {}))
-            .featureStates(new String[] {"none"})
-            .waitForCompletion(true);
+        CreateSnapshotRequest.of(
+            b ->
+                b.repository(snapshotRequest.repositoryName())
+                    .snapshot(snapshotRequest.snapshotName())
+                    .indices(snapshotRequest.indices())
+                    // ignoreUnavailable = false - indices defined by their exact name MUST be
+                    // present
+                    // allowNoIndices = true - indices defined by wildcards, e.g. archived, MIGHT BE
```

Comment:
I think this should be mapped to "[expand_wildcards"](https://www.elastic.co/guide/en/elasticsearch/reference/current/create-snapshot-api.html#create-snapshot-api-request-body) field in the json, but it's not present in the request from ElasticsearchClient